index.html

```diff
@@ -239,7 +245,7 @@ <h2 id="aboutTitle">About</h2>
 
         </section>
 
-
+    
```

Comment:
https://github.com/DamianDib9/averger/blob/317f3306df51d0ba24cc1acc2f9198b529bc68c0/index.html#L270-L394

- [OPTIONAL]As requested by the previous reviewer,  In this particular project, the section of the whole project along with the models should be created dynamically with Javascript. Great job so far however the section of the entire work must be removed from HTML, please address this 🚮